### DISCUSSION TITLE
extension libraries for SQLX etc.

### DISCUSSION THREAD

**Discussion by databrecht** [greyblake/nutype#131] [Ideas]
Are there any extension libraries that you are aware of to be able to insert/return nutypes from the database with nutype?
Or does the community typically manually implement it as explained here:
https://users.rust-lang.org/t/how-should-one-use-sqlx-with-newtype-pattern/73172

**Comment by greyblake** [comment]
Hi, sorry for the delayed reply (I apologize, I miss the message).

I have a middle-term plan to support derives for Diesel and SQLx.
